components/dashboard/src/whatsnew/MigrationPage.tsx

```diff
@@ -120,18 +117,18 @@ export function MigrationPage() {
     );
 }
 
-function useMigrateUserMutation() {
-    const invalidateOrgs = useOrganizationsInvalidator();
-    const { setUser } = useContext(UserContext);
+function useMarkMessageReadMutation() {
+    const { user, setUser } = useContext(UserContext);
 
     return useMutation<User, Error>({
         mutationFn: async () => {
-            const user = await getGitpodService().server.migrateLoggedInUserToOrgOnlyMode();
-            setUser(user);
-            return user;
-        },
-        onSuccess(updatedOrg) {
-            invalidateOrgs();
+            if (!user) {
+                throw new Error("No user");
+            }
+            let updatedUser = AdditionalUserData.set(user, { shouldSeeMigrationMessage: false });
+            updatedUser = await getGitpodService().server.updateLoggedInUser(updatedUser);
+            setUser(updatedUser);
```

Comment:
Would it make sense to also pro-actively clear out the `skip-migration` key from localStorage here too, or does that get handled somewhere else?